components/org.wso2.carbon.identity.oauth.endpoint/src/test/java/org/wso2/carbon/identity/oauth/endpoint/token/OAuth2TokenEndpointTest.java

```diff
@@ -347,11 +354,15 @@ public void testTokenErrorResponse(String errorCode, Object headerObj, int expec
         requestParams.put(OAuth.OAUTH_USERNAME, new String[]{USERNAME});
         requestParams.put(OAuth.OAUTH_PASSWORD, new String[]{"password"});
 
+        OAuthClientAuthnContext oAuthClientAuthnContext = new OAuthClientAuthnContext();
+        oAuthClientAuthnContext.setAuthenticated(true);
```

Comment:
In here also i can see the data provider passing params that reflect authorization errors so is setting the client authentication like this correct?

Reply:
Yeah, the way this units tests are written, won't affect the variable `isAuthenticated`